embedded.go

```diff
@@ -17,12 +19,20 @@ var (
 	configAuditReportsCRD []byte
 	//go:embed deploy/crd/clusterconfigauditreports.crd.yaml
 	clusterConfigAuditReportsCRD []byte
+	//go:embed deploy/crd/clustercompliancereports.crd.yaml
+	clusterComplianceReportsCRD []byte
+	//go:embed deploy/crd/clustercompliancedetailreports.crd.yaml
+	clusterComplianceDetailReportsCRD []byte
 	//go:embed deploy/crd/ciskubebenchreports.crd.yaml
 	kubeBenchReportsCRD []byte
 	//go:embed deploy/crd/kubehunterreports.crd.yaml
 	kubeHunterReportsCRD []byte
 	//go:embed  deploy/static/04-starboard-operator.policies.yaml
 	policies []byte
+
+	// compliance reports
```

Comment:
GoDoc starts with the name of the variable and is follow by a sentence explaining this property. If we are not explaining anything let's remove this comment